CMakeLists.txt

```diff
@@ -172,6 +173,12 @@ if (UNIX)
             install(FILES "dist/flatpak/input-leap-flatpak"
                     DESTINATION bin
                     PERMISSIONS OWNER_EXECUTE OWNER_WRITE OWNER_READ GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
+
+	    if (INPUTLEAP_WARN_ON_WAYLAND)
+                add_definitions(-DINPUTLEAP_WARN_ON_WAYLAND=1)
```

Comment:
Band indentation.

Reply:
Fixed in 875775d8. Also moved the block to a more logical place.